packages/utils/lib/axios.ts

```diff
@@ -37,7 +37,8 @@ if (hasHttpsProxy) {
 
 const config: AxiosRequestConfig = {
     httpAgent: httpAgent,
-    httpsAgent: httpsAgent
+    httpsAgent: httpsAgent,
+    ...(process.env['AXIOS_HTTP_AGENT'] ? JSON.parse(process.env['AXIOS_HTTP_AGENT']) : {})
```

Comment:
is it gonna be temporary? 
Also we should probably ensure the json parsing is successful before to assign it

Reply:
it's probably going to be temporary until I find the right configuration, but maybe it's going to be one configuration per service so not sure. 
you are right I'll add a try/catch to be extra cautious 👌🏻